Automattic/jetpack
# Admin Page: remove Themes card on Plans tab

Fixes #12366

#### Changes proposed in this Pull Request:

Let's remove the Themes card from the Plans tab for now.

**Before**

![](https://user-images.githubusercontent.com/1123119/57717308-302ccf80-7638-11e9-8760-b7716db49997.png)

**After**

![image](https://user-images.githubusercontent.com/426388/57723349-b8d35c00-7688-11e9-9d97-d25290ff0362.png)


#### Testing instructions:

* Go to Jetpack > Dashboard > Plans
* Does the plan display nicely? Do you not see the Themes card?

#### Proposed changelog entry for your changes:

* Admin Page: remove Themes card on Plans tab
